app/models/offer.rb

```diff
@@ -0,0 +1,6 @@
+class Offer < ApplicationRecord
+  validates :description, presence: true, length: { minimum: 5 }
+  validates :title, presence: true
+  validates :product, presence: true
+  validates :suggested_price, numericality: { only_integer: true }
+end
```

Comment:
it's time to add some associations. Add assosiation to product and to customer.

Reply:
Added two associations, according to the specs on Trello "that offer will be owned by the customer and by the product". I think :)

Reply:
Here's a hint: http://guides.rubyonrails.org/association_basics.html